api/src/main/proto/stellarstation/api/v1/groundstation/groundstation.proto

```diff
@@ -208,3 +226,69 @@ message DeleteUnavailabilityWindowRequest {
+  // The ID of the ground station that is opening a stream. The ID of a ground station can be found
+  // on the StellarStation Console page for the ground station.
+  string ground_station_id = 1;
+
+  // A request to send on the ground station stream. The request does not need to be filled on the
+  // first request to `OpenGroundStationStream`.
+  oneof Request {
+    SendTelemetryRequest send_telemetry_request = 2;
+
+    // An event that occurred while processing the stream. Satellite operators will often use these
+    // events for debugging and monitoring.
+    //
+    // - `command_sent` must be returned right after completing transmission of
+    //    commands received in a `SendSatelliteCommandsResponse`.
+    // - `plan_monitoring_event.ground_station_configuration` must be returned once at the beginning
+    //   of the execution of a plan and contains information about the actual configuration of the
+    //   ground station at the time.
+    // - `plan_monitoring_event.ground_station_state` must be sent continuously throughout the
+    //   plan and contains real-time monitoring of the hardware of the ground station.
+    // - `plan_monitoring_event.ground_station_event` must be sent once at each of the transitions
+    //   documented in `PlanLifecycleEvent` when executing a plan.
+    StreamEvent stream_event = 3;
+  }
+}
+
+// A request to send telemetry received from a satellite to a user.
+message SendTelemetryRequest {
+  // The ID of the plan the telemetry is being sent for.
+  string plan_id = 1;
+
+  // The telemetry being sent.
+  Telemetry telemetry = 2;
+}
+
+// A response for the `OpenGroundStationStream` method. The ground station must process a response
+// immediately when received on the stream.
+message GroundStationStreamResponse {
+  // The ID of the plan this response corresponds to. Unfilled if this response does not correspond
+  // to a plan.
+  string plan_id = 1;
+
+  // A unique ID identifying this response. If filled, any `StreamEvent` generated when handling
+  // this GroundStationStreamResponse` must have this value copied into
+  // `StreamEvent.request_id`. Currently, this includes
+  //
+  // - Returning `CommandSentFromGroundStation` after completing transmission of the commands in a
+  //   `SendSatelliteCommandsResponse`.
+  string response_id = 2;
+
+  // A response sent on the stream to be processed by the ground station. Currently only commands to
+  // send to a satellite will be returned to the ground station.
+  oneof Response {
+    // A response containing commands to send to the satellite.
+    SendSatelliteCommandsResponse send_satellite_commands_response = 3;
+  }
+}
+
+// A response containing a command to transmit to the satellite.
+message SendSatelliteCommandsResponse {
+  // The command frames to send to the satellite. All commands must be transmitted in sequence
+  // immediately. After all commands have been transmitted, telemetry receive will be immediately.
```

Comment:
I don't think the end of the sentence makes sense: 'telemetry receive will be immediately.'